README.md

```diff
@@ -204,7 +204,6 @@ OPTIONS:
         --solver-type <solver-type>
             Which style of solver to use. Can be one of: 'naive-solver' for the naive solver;
             'standard-solver' for mixed integer programming solver;
-            'fallback-solver' for a more conservative solver than the standard solver;
```

Comment:
I think this comment needs to also be updated in `main.rs` (unless we remove it there completely)

Reply:
nit: this section should stay in sync with the output from `cargo run -p driver -- --help`.

Reply:
It would be very fancy to add a travis check that greps for the output of `cargo run` in out README and fails if it's not in sync.